cmake/modules/SwiftSupport.cmake

```diff
@@ -38,6 +38,10 @@ function(add_swift_library library)
     endforeach()
   endif()
 
+  if(CMAKE_SYSTEM_NAME STREQUAL Android)
+    list(APPEND flags -sdk;${CMAKE_SYSROOT})
```

Comment:
It probably makes sense to always add the sysroot?

Reply:
Yeah, I think that we should always add the sysroot here.